l1-contracts/contracts/bridge/interfaces/IL2Bridge.sol

```diff
@@ -4,7 +4,13 @@ pragma solidity 0.8.24;
 
 /// @author Matter Labs
 interface IL2Bridge {
-    function finalizeDeposit(bytes32 _assetInfo, bytes calldata _data) external;
+    function finalizeDeposit(
```

Comment:
finalizeDeposit, with these variables should be part of the legacy bridge

Reply:
updated